### PR TITLE
new metadb derived table Holdings statements

### PR DESCRIPTION
**Pull request by natalyapik**
add new metadb derived table for holdings also added to runlist

### CODE REVIEW

**Review by jenmawe — CHANGES_REQUESTED** [review]
The only thing I saw was that in the Create Index at the bottom you have note instead of public_note. Once I changed that, the query ran.
- [] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

**Review by adoerrer — APPROVED** [review]
After the update query looks good, thank you! 

- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

**Review by jenmawe — CHANGES_REQUESTED** [review]
On line 42, just a change from note to public_note

**Review by jenmawe — APPROVED** [review]

File: sql_metadb/derived_tables/holdings_statements.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+WITH holdings AS (
+SELECT 
+    h.instanceid AS instance_id,
+    h.id AS holdings_id,
+    jsonb_extract_path_text(h.jsonb, 'hrid') AS holdings_hrid,
+    jsonb_extract_path_text(hs.jsonb, 'statement') AS holdings_statement,
+    jsonb_extract_path_text(hs.jsonb, 'note') AS public_note,
+    jsonb_extract_path_text(hs.jsonb, 'staffNote') AS staff_note,
+    hs.ordinality AS statement_ordinality
+    FROM folio_inventory.holdings_record h
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/holdings_statements.sql:16]: We are tending to use `FROM <table> AS <alias>`.  I will make this change.

File: sql_metadb/derived_tables/holdings_statements.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    hs.instance_id,
+    jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,   
+    hs.holdings_id,
+    hs.holdings_hrid,
+    hs.holdings_statement,
+    hs.public_note,
+    hs.staff_note, 
+    hs.statement_ordinality
+    FROM holdings AS hs
+    LEFT JOIN folio_inventory.instance i ON hs.instance_id = i.id
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/holdings_statements.sql:29]: And again here.  I will make the change.

**Pull request merged**